lib/SIL/PrettyStackTrace.cpp

```diff
@@ -73,3 +73,10 @@ void PrettyStackTraceSILFunction::printFunctionInfo(llvm::raw_ostream &out) cons
   if (SILPrintOnError)
     TheFn->print(out);
 }
+
+void PrettyStackTraceSILNode::print(llvm::raw_ostream &out) const {
+  if (currentSILNode != nullptr)
+    out << "Visiting SIL Node " << *currentSILNode;
+
+  PrettyStackTraceSILFunction::print(out);
```

Comment:
This part doesn't seem useful. You'll never be serializing a SIL instruction without already being within the `PrettyStackTraceSILFunction` of its parent.